pom.xml

```diff
@@ -1432,8 +1443,8 @@
 
     <!-- NOTE: When updating GeoTools, check which version
     of Postgres is used and update pg.version if needed. -->
-    <geotools.version>16.0</geotools.version>
-    <pg.version>9.4.1211</pg.version>
+    <geotools.version>22.3</geotools.version>
```

Comment:
Note `23-RC` is out now.

Reply:
23.0 will be out this week.

Reply:
Updated to 23.0 on new GeoSolution repository